packages/icons/src/img/sprite.svg

```diff
@@ -2031,5 +2031,8 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 64 64" id="lp-icon-osmo">
```

Comment:
The viewbox isn't the expected 20x20. Is this expected @matthewferry?

Reply:
i took this from gonfalon…

does this matter? can we fix later?

Reply:
Hard to tell. Will let design approve the chromatic build to be safe.

Reply:
Given the viewbox in SVG is internal in that it controls the SVG viewport, I don't see the risk, but we can wait.

Reply:
view box should be '20 20'. hang tight. I'll get you the svg :)